AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/README.md

```diff
@@ -1,6 +1,6 @@
-# `Intel® TensorFlow* Model Zoo Inference With FP32 Int8` Sample
+# `Intel® Reference models TensorFlow* Inference With FP32 Int8` Sample
 
-The `Intel® TensorFlow* Model Zoo Inference With FP32 Int8` sample demonstrates how to run ResNet50 inference on pretrained FP32 and Int8 models included in the Model Zoo for Intel® Architecture.
+The `Intel® Reference models TensorFlow* Inference With FP32 Int8` sample demonstrates how to run ResNet50 inference on pretrained FP32 and Int8 models included in the Reference models for Intel® Architecture.
```

Comment:
the `Intel® AI Reference models for TensorFlow* Inference` sample?

Reply:
addressed it accordingly